cmd/mr_show.go

```diff
@@ -162,80 +159,13 @@ WebURL: %s
 		mr.Author.Username, milestone, labels, mr.WebURL)
 }
 
-func printMRDiscussions(discussions []*gitlab.Discussion, since string, mrNum int) {
-	NewAccessTime := time.Now().UTC()
-
-	mrEntry := fmt.Sprintf("mr%d", mrNum)
-	// if specified on command line use that, o/w use config, o/w Now
-	var (
-		CompareTime time.Time
-		err         error
-		sinceIsSet  = true
-	)
-	CompareTime, err = dateparse.ParseLocal(since)
-	if err != nil || CompareTime.IsZero() {
-		CompareTime = getMainConfig().GetTime(CommandPrefix + mrEntry)
-		if CompareTime.IsZero() {
-			CompareTime = time.Now().UTC()
-		}
-		sinceIsSet = false
-	}
-
-	// for available fields, see
-	// https://godoc.org/github.com/xanzy/go-gitlab#Note
-	// https://godoc.org/github.com/xanzy/go-gitlab#Discussion
-	for _, discussion := range discussions {
-		for i, note := range discussion.Notes {
-
-			// skip system notes
-			if note.System {
-				continue
-			}
-
-			indentHeader, indentNote := "", ""
-			commented := "commented"
-			if !time.Time(*note.CreatedAt).Equal(time.Time(*note.UpdatedAt)) {
-				commented = "updated comment"
-			}
-
-			if !discussion.IndividualNote {
-				indentNote = "    "
-
-				if i == 0 {
-					commented = "started a discussion"
-				} else {
-					indentHeader = "    "
-				}
-			}
-
-			printit := color.New().PrintfFunc()
-			printit(`
-%s-----------------------------------`, indentHeader)
-
-			if time.Time(*note.UpdatedAt).After(CompareTime) {
-				printit = color.New(color.Bold).PrintfFunc()
-			}
-			printit(`
-%s%s %s at %s
-
-%s%s
-`,
-				indentHeader, note.Author.Username, commented, time.Time(*note.UpdatedAt).String(),
-				indentNote, note.Body)
-		}
-	}
-
-	if sinceIsSet == false {
-		config.WriteConfigEntry(CommandPrefix+mrEntry, NewAccessTime, "", "")
-	}
-}
-
 func init() {
 	mrShowCmd.Flags().BoolP("no-markdown", "M", false, "Don't use markdown renderer to print the issue description")
 	mrShowCmd.Flags().BoolP("comments", "c", false, "Show comments for the merge request")
 	mrShowCmd.Flags().StringP("since", "s", "", "Show comments since specified date (format: 2020-08-21 14:57:46.808 +0000 UTC)")
 	mrShowCmd.Flags().BoolVarP(&mrShowPatch, "patch", "p", false, "Show MR patches")
 	mrShowCmd.Flags().BoolVarP(&mrShowPatchReverse, "reverse", "", false, "Reverse order when showing MR patches (chronological instead of anti-chronological)")
+	mrShowCmd.Flags().BoolVarP(&mrShowColorDiff, "color-diff", "", false, "Show color diffs in comments")
```

Comment:
I would say color should be the default, and if anything the option should be one that disables color. That just feels right for a modern CLI. The ci logs/traces have color as a part of them as well.

Reply:
I've flipped the option to --no-color-diff, and set to "false" by default.  ie) the color is on by default.